docs/en/changes/changes.md

```diff
@@ -24,6 +24,7 @@
 * Remove unnecessary annotations and functions from Meter Functions.
 * Add `max` and `min` functions for MAL down sampling.
 * Fix critical bug of uncontrolled memory cost of TopN statistics. Change topN group key from `StorageId` to `entityId + timeBucket`.
+* Fix wrong timeBucket in topN group key.
```

Comment:
```suggestion
```

You don't need this. This bug never released.

Reply:
OK.